babel.config.js

```diff
@@ -4,18 +4,9 @@ module.exports = {
     [
       'transform-imports',
       {
-        '@material-ui/core/styles': {
+        '@material-ui/core': {
```

Comment:
Are you 100% sure this is not capturing @material-ui/core/styles  @kilemensi  ?

Reply:
I ran into issues where it would match `core/styles` or maybe that was an oversight.